tests/test_suites/SanityChecks/test_prometheus.sh

```diff
@@ -0,0 +1,29 @@
+if ! ldd $(whereis sfsmaster | awk '{print $2}') | grep prometheus; then
+	echo "Prometheus client isn't linked with master, skipping test..."
+	test_end
+fi
```

Comment:
Nice, but what happens if there is a master without Prometheus support installed for instance in /usr/sbin? Probably the testing framework contains a direct way to know where are the testing binaries.